extensions/blocks/tiled-gallery/index.js

```diff
@@ -128,6 +128,10 @@ const blockAttributes = {
 		default: 'none',
 		type: 'string',
 	},
+	roundedCorners: {
```

Comment:
`borderRadius` could be another name and match CSS rule, but figured this is more self-explanatory.